R/survival.R

```diff
@@ -1,89 +1,75 @@
+#' @param ..surv_juv_rear_contact_points coefficient for \cpde{contact_points} variable, source: calibration
+#' @param ..surv_juv_rear_prop_diversions coefficient for \code{prop_diversions} variable, source: calibration
+#' @param ..surv_juv_rear_total_diversions coefficient for \code{total_diversions} variable, source: calibration
+#' @param .stranded coefficient for \code{stranded} variable, source: \href{#}{USFWS (2006) and CDWR (2006)}
+#' @param .medium parameter for medium sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
+#' @param .large parameter for large sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
+#' @param .floodplain parameter for floodplain rearing benefit, source: \href{https://dsm-docs.s3-us-west-2.amazonaws.com/SOMMER_T-SDWA+180+Floodplain+rearing+of+juvenile+chinook+salmon+evidence+of+enhanced+growth+and+survival+.pdf}{Sommer et al. (2001)}
 #' @source IP-117068
 #' @export
 surv_juv_rear <- function(max_temp_thresh, avg_temp_thresh, high_predation,
                           contact_points, prop_diversions, total_diversions,
                           stranded, weeks_flooded,
-                          betas = c(`2nd calibration adjustment` = -0.6558315,
-                                    `average temperature` = -0.717,
-                                    predation = -0.122,
-                                    `contact points` =  0.0194795,
-                                    `contact points scaler` = -0.189,
-                                    `proportion diverted` = 0.1000000,
-                                    `proportion diverted scaler` = -3.51,
-                                    `total diverted` = 0.3000000,
-                                    `total diverted scaler` = -0.0021,
-                                    stranded = -1.939,
-                                    medium = 1.48,
-                                    large = 2.223,
-                                    `floodplain habitat` = 0.47,
-                                    `surival adjustments` = 1)){
+                          ..surv_juv_rear_int = -0.6558315,
+                          .avg_temp_thresh = -0.717,
+                          .high_predation = -0.122,
+                          ..surv_juv_rear_contact_points =  -0.00368,
+                          ..surv_juv_rear_prop_diversions = -0.351,
+                          ..surv_juv_rear_total_diversions = -0.00063,
+                          .stranded = -1.939,
+                          .medium = 1.48,
+                          .large = 2.223,
+                          .floodplain = 0.47,
+                          .surival_adjustments = 1){ # TODO figure out how we want to add in scenario surv_adjustments
```

Comment:
add `.surival_adjustments` to param list in docs, just as a place holder for now

Reply:
Added